client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -25,270 +25,259 @@ import { recordGoogleEvent } from 'calypso/state/analytics/actions';
 import { getReaderTeams } from 'calypso/state/teams/selectors';
 import SubscriptionManagementBackButton from '../subscription-management-back-button';
 
-class NotificationSubscriptions extends Component {
-	handleClickEvent( action ) {
-		return () => this.props.recordGoogleEvent( 'Me', 'Clicked on ' + action );
-	}
+const NotificationSubscriptions = ( props ) => {
+	const dispatch = useDispatch();
+	const teams = useSelector( getReaderTeams );
+	const isAutomattician = isAutomatticTeamMember( teams );
+	const { translate } = props;
 
-	handleFocusEvent( action ) {
-		return () => this.props.recordGoogleEvent( 'Me', 'Focused on ' + action );
-	}
+	const handleClickEvent = useCallback(
```

Comment:
Those `useCallback` seem unnecessary. Preventing the re-render of a single link or a button is just not worth the memoization overhead IMHO. Same for the rest of the `dispatch()` calls wrapped in `useCallback` around here.